src/table/handle-data.js

```diff
@@ -28,7 +31,8 @@ export default async function manageData(model, layout, pageInfo) {
   ]);
   const matrix = dataPages[0].qMatrix;
 
-  const columns = columnorder.map((c) => getColumnInfo(layout, c));
+  // using filter to remove hidden columns (represented with false)
+  const columns = columnorder.map((c) => getColumnInfo(layout, c)).filter(Boolean);
```

Comment:
I look at the filter(Boolean) trick. It is interesting. It is very good that you write comments here.

Reply:
Agreed, it is not common syntax. Found it on stack overflow, someone else being the smart one here 🙃